schemas/theme/setting.json

```diff
@@ -0,0 +1,422 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "type": {
+      "oneOf": [
+        {
+          "const": "article",
+          "description": "A setting of type article outputs an article picker field that's automatically populated with the available articles for the store. You can use these fields to capture an article selection, such as the article to feature on the homepage.",
+          "markdownDescription": "A setting of type `article` outputs an article picker field that's automatically populated with the available articles for the store. You can use these fields to capture an article selection, such as the article to feature on the homepage.\n\n---\n\n[Shopify reference](https://shopify.dev/docs/themes/architecture/settings/input-settings#article)"
+        },
```

Comment:
Turns out that the way to have a "documented enum" is to have a `oneOf` rule of `const`. Does the same thing but each const can be documented.

Reply:
Makes sense.
Is this something in the official docs or did you do some detective work to find out about this?

I tried to find some explanations on my own and found some info on [stackoverflow](https://stackoverflow.com/questions/64233370/in-json-schema-how-to-define-an-enum-with-description-of-each-elements-in-the-e) and [github](https://github.com/json-schema-org/json-schema-spec/issues/1386)

Reply:
(For documentation purposes): Discovered it from trial and error. It's much more verbose, but also probably easier to maintain in the new setup.